data_steward/cdr_cleaner/clean_cdr.py

```diff
@@ -46,9 +47,71 @@
 import cdr_cleaner.manual_cleaning_rules.remove_operational_pii_fields as operational_pii_fields
 import cdr_cleaner.manual_cleaning_rules.update_questiona_answers_not_mapped_to_omop as map_questions_answers_to_omop
 from constants.cdr_cleaner.clean_cdr import DataStage as stage
+from constants.cdr_cleaner import clean_cdr as cdr_consts
 
 LOGGER = logging.getLogger(__name__)
 
+PROJECT_ID = 'project_id'
+DATASET_ID = 'dataset_id'
+SANDBOX_DATASET_ID = 'sandbox_dataset_id'
+
+
+class CleanCdrIllegalArgumentError(ValueError):
+    pass
+
+
+def add_module_info_decorator(query_function,
+                              project_id,
+                              dataset_id,
+                              sandbox_dataset_id=None):
+    """
+    A decorator for adding the module information to the list of query dictionaries generated by the cleaning rules
+    :param query_function: a function that generates a list of query dictionaries
```

Comment:
For the purpose of automated documentation, there should be a blank line after the one line summary sting.

Reply:
Added